packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -871,15 +932,15 @@ export class MgtPerson extends MgtTemplatedComponent {
     const details: TemplateResult[] = [];
 
     if (this.view > ViewType.image) {
+      const text = this.getTextFromProperty(person, this.line1Property);
       if (this.hasTemplate('line1')) {
         // Render the line1 template
         const template = this.renderTemplate('line1', { person });
         details.push(html`
-          <div class="line1" @click=${() => this.handleLine1Clicked()}>${template}</div>
+          <div class="line1" @click=${() => this.handleLine1Clicked()} aria-label="${text}">${template}</div>
```

Comment:
If we have a click behavior, should we have an aria-role that hints that this element is clickable?
This comment applies to all the clickable lines.